common/changes/@itwin/imodel-browser-react/DanishM-itwinui3_x_2023-11-20-07-14.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/imodel-browser-react",
+      "comment": "Updated itwinui-react to 3.x",
+      "type": "major"
```

Comment:
As discussed internally, please change these (and other ones) to "minor" only.

Reply:
Updated to minor